### ISSUE TITLE
FTPEntryType not exported

### ISSUE THREAD

**Issue by dennismuench** [salim-lachdhaf/dartFTP#6]
`FTPEntryType` is not exported, so it is not possible to distinguish between files and folders using `FTPEntry.type`.

Dirty workaround: Using the auto-assigned index of the enum:

```dart
enum FTPEntryType {
  FILE, // 0
  DIR, // 1
  LINK, // 2
  UNKNOWN // 3
}
```

Eg:

```dart
final entries = ftp.listDirectoryContent();

entries.forEach((entry) {
  if (entry.type.index == 0) {
    // Entry is file
  }
  else if (entry.type.index == 1) {
    // Entry is directory
  }
  ...
}
```

**Comment by TheKingDave** [comment]
Another workaround is to add an import to `ftp_entry.dart`.
Eg:
```dart
import 'package:ftpconnect/src/ftp_entry.dart';
```

**Comment by salim-lachdhaf** [comment]
issue fixed in 2.0.6
